extensions/amp-analytics/0.1/test/test-analytics-root.js

```diff
@@ -739,23 +763,38 @@ describes.realWin(
       });
 
       it('should find all elements by selector', async () => {
+        const spy = env.sandbox.spy(user(), 'warn');
         const child2 = win.document.createElement('child');
         const child3 = win.document.createElement('child');
+        const child4 = win.document.createElement('child');
         // Parent child attached to parent doc should not be captured
         const parentChild = env.parentWin.document.createElement('child');
         body.appendChild(child2);
         body.appendChild(child3);
+        body.appendChild(child4);
         env.parentWin.document.body.appendChild(parentChild);
         child.classList.add('myClass');
         child2.classList.add('myClass');
+        child4.classList.add('myClass');
```

Comment:
can we remove this? Why we need child4?

Reply:
Child4 has no data-vars-* attribute, so it's testing that it doesn't get picked up (I'll add a comment).

Reply:
My bad! Complete missed that part. 

I understand this is the test case for embed and it's duplicate version of the test above. But this unit test covers
1.selector select all elements
2. select element within the doc
3. data-vars

Maybe consider breaking it to multiple unit tests next time.

Reply:
Done :)